docs/CSBR.md

```diff
@@ -401,13 +403,13 @@ For EV Code Signing Certificates, use of documents, data, and previous validatio
 
 CAs MUST not issue new or replacement Code Signing Certificates to an entity that the CA determined intentionally signed Suspect Code. The CA MUST keep meta-data about the reason for revoking a Code Signing Certificate as proof that the Code Signing Certificate was not revoked because the Applicant was intentionally signing Suspect Code.
 
-CAs MAY issue new or replacement Code Signing Certificates to an entity who is the victim of a documented Takeover Attack, resulting in either a loss of control of their code-signing service or loss of the Private Key associated with their Code Signing Certificate.
+CAs MAY issue new or replacement Code Signing Certificates to an entity who is the victim of a documented Takeover Attack, resulting in a loss of control of the Private Key associated with their Code Signing Certificate.

```

Comment:
Since the ballot now likely won't be effective until after the HSM deadline, perhaps we go ahead and remove the Takeover Attack verbiage as previously agreed?  As we discussed, that language was only there as a compensating control because we were unable to mandate HSM usage at the time the language was written, c. 2015.